src/neo/IO/Json/JObject.cs

```diff
@@ -16,8 +16,9 @@ public class JObject
         {
             get
             {
-                Properties.TryGetValue(name, out JObject value);
-                return value;
+                if (Properties.TryGetValue(name, out JObject value))
+                    return value;
+                return null;
```

Comment:
Any difference?

Reply:
The return is the same, but I think that we always use `TryGetValue` like that